pkg/kubelet/kubelet.go

```diff
@@ -492,8 +494,6 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 	klet.podManager = kubepod.NewBasicPodManager(kubepod.NewBasicMirrorClient(klet.kubeClient))
 
 	if kubeCfg.RemoteRuntimeEndpoint != "" {
-		kubeCfg.ContainerRuntime = "remote"
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L196 should also be updated since it not override `--container-runtime` any more.